src/main/java/com/powsybl/ws/commons/computation/ComputationException.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.ws.commons.computation;
+
+/**
+ * @author Joris Mancini <joris.mancini_externe at rte-france.com>
+ */
+public class ComputationException extends RuntimeException {
+    public ComputationException(String message, Throwable cause) {
```

Comment:
probably better to add the constructor without cause right now

Reply:
done

Reply:
drawback of adding unused code, it's not covered by test ^^